modules/ROOT/pages/6.0-release-notes-plugins.adoc

```diff
@@ -0,0 +1,300 @@
+= TinyMCE 6.0 Plugin changes
+:navtitle: TinyMCE 6.0 Plugin changes
+:description: TinyMCE 6.0 Plugin changes
+:keywords: releasenotes, plugin
+
+This section documents the improvements, changes, and removals made to {productname} 6.0 Plugins.
+
+It also documents the the plugin loading change introduced in {productname} 6.0.
+
+* xref:plugins[Plugins]
+* xref:plugin-loading-format-change[Plugin loading format change]
+* xref:new-and-improved-plugins][New and improved Plugins]
+* xref:changed-apis[Changed APIs]
+* xref:removed-or-deprecated-plugins[Removed or deprecated Plugins]
+
+// tag::plugins[]
+[[plugins]]
+== Plugins
+
+[[plugin-loading-format-change]]
+=== Plugin loading format change
+
+{productname} 6.0 allows plugin loading in any of three formats:
+
+. An array of strings:
++
+`plugins: [ 'advlist', 'list', 'image', 'help', 'wordcount' ]`
++
+. A space-separated string:
++
+`plugins: 'advlist list image help wordcount'`
++
+. A comma-separated string:
++
+`plugins: 'advlist,list,image,help,wordcount'`
+
+Previously, two other formats were allowed: comma-separated strings with spaces and arrays of space-separated strings.
+
+Neither of these formats are allowed in {productname} 6.0.
+
+
+[[new-and-improved-plugins]]
+=== New and improved plugins
+
+[[imagetools]]
+==== EditImage
+
+_EditImage_, a https://tiny.cloud/tinymce/features/#productivity[Premium plugin], replaces the now-deprecated and removed _ImageTools_ plugin. The `imagetools` plugin is no longer part of the Core open source editor.
+
+An `editimage` option was also renamed: `editimage_proxy` is now `editimage_proxy_service_url`, making it easier to configure when accessing https://tiny.cloud/docs/enterprise/server/[Tiny-specific services].
+
+`editimage` also offers a new export plugin option: `export_image_proxy_service_url`.
+
+NOTE: `editimage_proxy` (the renamed equivalent to the previous `imagetools_proxy`) is still available as an alternative configuration option for setting up image proxy functionality without using Tiny services.
+
+
+[[emoticons]]
+==== Emoticons
+
+The `emoticon` plugin now uses the more accurate word, _Emoji_, in its tool bar, menu item and dialog box.
+
+In previous versions, the word presented in these parts of the plugin’s user-interface was _Emoticon_.
+
+NOTE: The plugin, itself, is still called Emoticons. And the plugin description and the plugin’s configuration and Help dialogs still reflect this.
+
+
+[[table-of-contents]]
+==== Table of contents
+
+There are no end-user facing changes between the previous `toc` plugin and the `tableofcontents` plugin. However, the `tableofcontents` plugin now uses https://terser.org[terser] to improve performance.
```

Comment:
```suggestion
There are no end-user facing changes between the previous `toc` plugin and the `tableofcontents` plugin.
```
This statement is incorrect and has nothing to do with performance, so we should just remove it.